cms/djangoapps/contentstore/utils.py

```diff
@@ -1565,6 +1568,7 @@ def get_library_context(request, request_is_json=False):
             'courses': [],
             'libraries_enabled': libraries_v1_enabled(),
             'show_new_library_button': user_can_view_create_library_button(request.user) and request.user.is_active,
+            'show_new_library_v2_button': user_can_create_library(request.user) and request.user.is_active,
```

Comment:
Question: Isn't the `is_active` check redundant here?

Reply:
Yes! Good catch!
Removed here: 30c6d06ef562f0790a08bf38fe3bb82ebaabaf58